src/argilla/server/apis/v0/handlers/users.py

```diff
@@ -47,6 +50,13 @@ async def whoami(request: Request, current_user: User = Security(auth.get_user,
     return current_user
 
 
+@router.get("/users", response_model=List[User], response_model_exclude_none=True)
+def list_users(*, db: Session = Depends(get_db), current_user: User = Security(auth.get_user, scopes=[])):
+    users = accounts.list_users(db)
+
+    return parse_obj_as(List[User], users)
```

Comment:
We can go with this response but in the next step, we should change the response mode, even if the endpoint has no support for pagination, just to simplify upgrades once we include support for it.

Reply:
Sounds great.